app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -929,13 +930,8 @@ private void showClearAllNotesDialog() {
     }
 
     builder.setMessage(this.getString(R.string.delete_notes_confirmation_msg))
-        .setNeutralButton(this.getString(R.string.dismiss), new DialogInterface.OnClickListener() {
-          @Override
-          public void onClick(DialogInterface dialog, int which) {
-            // Do nothing
-          }
-        })
-        .setPositiveButton(this.getString(R.string.delete), new DialogInterface.OnClickListener() {
+        .setNegativeButton(this.getString(R.string.cancel), null) // Do nothing for 'Cancel' button
```

Comment:
A couple of notes:

- `this` is redundant and I wouldn't use it, some people like the explicitness but I don't think this scenario is confusing enough to warrant it
- `getString` is not needed, `setXXXXButton` all have stringId variants
- the android platform supplies many strings (and means we won't have to translate them) located at android.R.string.* , I believe they have a cancel

Reply:
Refactored [here](https://github.com/kiwix/kiwix-android/pull/1198/commits/77c30ac518ae9a2f872e6ca971917a64702ed323) along with in the previous commits
I'm using the platform string for "Cancel" and app-resource string for "Yes" (whose translations exist)